test/src/de/dkfz/b080/co/knowledge/metadata/sampleextractorstrategies/SampleFromFilenameExtractorVersionTwoTest.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018 German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ).
+ *
+ * Distributed under the MIT License (license terms are at https://github.com/DKFZ-ODCF/COWorkflowsBasePlugin/LICENSE.txt).
+ */
+package de.dkfz.b080.co.knowledge.metadata.sampleextractorstrategies
+
+import de.dkfz.roddy.RoddyTestSpec
+
+class SampleFromFilenameExtractorVersionTwoTest extends RoddyTestSpec {
```

Comment:
Did you consider to reuse this as VersionOne test. I mean just remove the tests that don't work. Or better adapt them to the actual output that you get. This would server well as documentation of the differences and supports us in bugfixing to identify change that affect the old version one parser.

Eventually, the version 2 is also meant as some kind of half-compatible substitute to version 1 and I can imagine many of these tests here also work for version 1.

Reply:
Did that and adapted the tests.